app/javascript/javascript/singleSession.js

```diff
@@ -5,21 +5,27 @@
 // The same for the empty point at the end.
 export const measurementsToTimeWithExtremes = ({ measurements, times }) => {
   const { start, end } = times;
-  return {
+  const ms = {
```

Comment:
I don't really like this name, even though it can be easily derived what this var is from the context

Reply:
I'm happy to change that if you have a better name :)

Reply:
well, what does `ms` stand for here? (sorry for responding so late!)

Reply:
I though Github glitched 😄 

Stands for measurements (`ms` plural of `m` where `m` stands for measurement). I guess I would have used `measurements` if that name was not already used in the arguments.

But I won't change it now 😉